### ISSUE TITLE
Secure your credentials cannot be undone

### ISSUE THREAD

**Issue by kernal64** [marcelklehr/floccus#788]
**Describe the bug**
I secured my credentials but changed my mind for my stationary PC. After looking for it, I found no option to remove the password, so I tried to enter an empty password but on next start, floccus still needs to be unlocked with the now empty password.

**To Reproduce**
1. Go to 'Secure your credentials'
2. Click on 'Set password for floccus'
3. Restart browser
4. See floccus is locked

**Expected behavior**
With no/blank password, floccus should not be locked.

**Desktop (please complete the following information):**
 - OS: Manjaro Linux (rolling)
 - Browser: Brave Browser
 - Browser Version: 1.18.78 Chromium: 87.0.4280.141 (Offizieller Build) unknown (64-Bit)
 - Floccus version: V4.3.0
 - Floccus sync method: nextcloud folders

**Server (please complete the following information):**
 - OS: _docker_
 - Nextcloud version: 20.0.4
 - Bookmarks app version: 4.0.5

Note: Please write down the actual version numbers instead of writing 'latest'.

**Debug log**
It is often useful to provide a debug log file along with the issue. You can obtain a debug log of the most recent sync runs by clicking 'Debug log' in the bottom right corner of the floccus panel in your browser.

Note: As your debug log usually contains large parts of your bookmark data, instead of attaching it publicly to the issue here, I recommend uploading it privately at the following URL https://klehranlage.net/index.php/s/7nn82NSBsfQKdsJ

- [ ] Debug log provided

**Additional context**

**Comment by marcelklehr** [comment]
After having entered a passphrase, when you click on "Your credentials are secure" you should be able to click on the "Remove passphrase" button.

**Comment by kernal64** [comment]
Ooops, to be honest, when I first tested floccus, I just "secured" the credentials with an empty passphrase without exctly knowing what I'm securing here. I thought, I could change the passphrase later. The remove button appeared, after I changed the empty passphrase to something. So it's just not possible to remove an empty passphrase I guess.

Nevertheless, nice tool. My test-phase finished succesfully and floccus now moves into production phase! ;-)
